2016-01-31 lacks snapshots for any BSD system, skip them to use 2015-04-02.

  gmake[2]: Entering directory '.../cargo-0.9.0'
  /usr/local/bin/python2.7 src/etc/dl-snapshot.py x86_64-unknown-freebsd
  Traceback (most recent call last):
    File "src/etc/dl-snapshot.py", line 57, in <module>
      raise Exception("no snapshot for the triple '%s'" % triple)
  Exception: no snapshot for the triple 'x86_64-unknown-freebsd'

--- src/snapshots.txt.orig	2016-04-08 20:31:50 UTC
+++ src/snapshots.txt
@@ -1,3 +1,6 @@
+2016-04-10
+  freebsd-x86_64 8c459cbdc890cedb43c0952ad7f1ef0b7fcbe535
+
 2016-03-21
   linux-i386 ac401c16ff53e0c51b88707579b4f95d7d4c4763
   linux-x86_64 84266cf626ca4fcdc290bca8f1a74e6ad9e8b3d9
